migrations/versions/1010_remove_unused_fields.py

```diff
@@ -0,0 +1,36 @@
+"""Remove unused fields
+
+Revision ID: 1010
+Revises: 1000
+Create Date: 2017-10-04 15:14:48.532073
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+
+# revision identifiers, used by Alembic.
+revision = '1010'
+down_revision = '1000'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Again, would be nice to remove these comments to show the auto-generated stuff has been sanity-checked by a human.